js/script.js

```diff
@@ -29,53 +29,90 @@
 		sharesChart;
 
 	$(document).ready(function () {
-		var x = document.getElementById('rambox');
-		x.style.backgroundColor = OCA.Theming  ? OCA.Theming.color : 'rgb(54, 129, 195)'; 
+		var rambox = document.getElementById('rambox');
+		rambox.style.backgroundColor = OCA.Theming ? OCA.Theming.color : 'rgb(54, 129, 195)';
 
-		var x = document.getElementById('swapbox');
-		x.style.backgroundColor = 'rgba(100, 100, 100, 0.8)';
+		var swapbox = document.getElementById('swapbox');
+		swapbox.style.backgroundColor = 'rgba(100, 100, 100, 0.8)';
 
-		resizeSystemCharts();
-		updateActiveUsersStatistics();
-		updateShareStatistics();
+		var updateTimer = setInterval(updateInfo, 300);
```

Comment:
Please remove this line. A few lines lester there should be a setTimeout call starting the updateInfo process.

Reply:
Please lets get this PR in and fix stuff later :see_no_evil: